game/Board.ts

```diff
@@ -93,17 +94,23 @@ export class Board {
             pl,
             { r, c },
             resource,
+            cityId,
             0,
             0,
-            Tile.NORMAL_TILE
+            Tile.CAPITAL_TILE
         );
 
         this.setTile(tl);
 
         this.playerCapitals.set(pl.bjjPub.serialize(), { r, c });
 
         // Update the merkle root on-chain.
-        await nStates.spawn(this.t[r][c].hash(), nullifier);
+        await nStates.spawn(
```

Comment:
oh is enclave supposed to call this? only the client should call contract functions, as per comment above